packages/TorneloScoresheet/src/hooks/appMode/arbiterTablePairingState.ts

```diff
@@ -31,5 +35,30 @@ export const makeUseArbiterTablePairingState =
       });
     };
 
-    return [appModeState, { goToTablePairingMode }];
+    const goBackToPairingSelectionMode = async (): Promise<Result<string>> => {
+      const pairings = await getStoredPairingList();
+      if (pairings) {
+        setAppModeState({
+          mode: AppMode.PairingSelection,
+          pairings,
+          games: pairings.length,
+        });
+        return succ('');
+      } else {
+        return fail(
+          'Error loading pairings, please go back to the PGN screen instead',
+        );
+      }
+    };
```

Comment:
Early return on the error case would be better to avoid the else and one level of indentation. But this is probably a nitpick, so feel free to ignore haha